### PR TITLE
Update React Native 0.29

### PR DESCRIPTION
**Pull request by dstnation**
React Native v0.28 `StatusBarIOS` is deprecated.

Android showing error, now
undefined is not a function(evaluating)
'StatusBarIOS.setStyle('defalut')')

Fix :
StatusBarIOS.setStyle to StatusBar.setBarStyle

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by nirzari18** [comment]
@dstnation Thank you! This was great help. I hope it gets merged soon...

File: js/tabs/F8TabsView.ios.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ class F8TabsView extends React.Component {
   };
 
   componentDidMount() {
-    StatusBarIOS && StatusBarIOS.setStyle('light-content');
+    StatusBar && StatusBar.setBarStyle('light-content');
   }
```
Comment (orta) [inline js/tabs/F8TabsView.ios.js]: Just started making this PR - thanks @dstnation 👍 don't have to do it now

**Pull request closed without merging**